.github/workflows/build.yml

```diff
@@ -0,0 +1,57 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
```

Comment:
All branches?

Reply:
It will trigger under these conditions:
- If the branch is a pull request which targets `master`
- Pushed directly to `master`.

I think it's pretty fine as is.